osu.Game/Screens/Edit/Setup/ResourcesSection.cs

```diff
@@ -125,17 +128,17 @@ public bool ChangeAudioTrack(FileInfo source)
             return true;
         }
 
-        private void backgroundChanged(ValueChangedEvent<FileInfo> file)
+        private void backgroundChanged(ValueChangedEvent<FileInfo?> file)
         {
-            if (!ChangeBackgroundImage(file.NewValue))
+            if (!ChangeBackgroundImage(file.NewValue!))
```

Comment:
`file.NewValue` should be explicitly null-checked rather than null-silenced with `!`. `ChangeBackgroundImage()` cannot cope with a `null`.

Same goes for `audioTrackChanged()` below.